compiler/plugins/target/ROCM/builtins/tuning/test/spec_gfx942.mlir

```diff
@@ -4,13 +4,23 @@
 // RUN:   --iree-codegen-notify-transform-strategy-application \
 // RUN:   --verify-diagnostics %s | FileCheck %s
 
+// RUN: iree-opt --split-input-file --iree-gpu-test-target=mi300x@hip \
+// RUN:   --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-hal-configure-target-executable-variants{target=rocm})))" \
+// RUN:   --iree-codegen-enable-default-tuning-specs \
+// RUN:   --iree-codegen-notify-transform-strategy-application \
```

Comment:
I missed when this flag was added, but it should be named `--iree-codegen-test` and/or be hidden or something. Not for this PR though.